java/server/src/org/openqa/selenium/grid/graphql/selenium-grid-schema.graphqls

```diff
@@ -14,6 +13,8 @@ enum Status {
 type Node {
   id: ID!
   uri: Uri!
+  host: String!
```

Comment:
This is duplicating information from the `uri` field. Please remove `host` and `port`